dev_guide/projects.adoc

```diff
@@ -194,3 +194,98 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+ifdef::openshift-online[]
+[[project-collaboration-in-online-pro]]
+== Project Collaboration in {product-title} Pro
+
+Every {product-title} Pro account has the ability to add up to 50 collaborator
+users per subscription. These collaborator users are granted cluster access from
+{product-title} Pro account subscribers so that they can collaborate on projects
+hosted with {product-title}. This allows multiple users to gain access to
+projects under a single subscription, without having to pay a monthly fee for
+every account.
+
+[NOTE]
+====
+Existing {product-title} Pro users do not need to be added as collaborators.
+====
+
+Collaboration is not available in {product-title} Starter.
+
+[[collaboration-restrictions]]
+=== Collaboration Restrictions
+
+Collaborators only have access to the specific projects (on clusters owned by
+the {product-title} Pro subscriber) that they are granted access to. They can
+not add projects of their own ownership. Collaborators can only add resources.
+
+[[project-collaboration-adding-collaborators]]
+=== Adding Collaborators
+{product-title} Pro subscribers can add collaborators by following these steps:
+
+. Each user you want to add as a collaborator must create a free account at
+ link:https://developers.redhat.com/[developers.redhat.com]. Once your
+ collaborator has confirmed their Red Hat Developers account, you can add them
+ to your subscription.
+
+ . Each collaborator must sign into
+link:https://developers.redhat.com/[developers.redhat.com] and click on their
+name in the upper-right corner to access their account details. Make note of the
+*Red Hat Login ID* on this page, as it is the user name you will be required to
+enter in order to associate the collaborator with your subscription.
+
+. Sign in to link:https://manage.openshift.com[manage.openshift.com] and click on
+*Manage Subscription* under the cluster you want to add the collaborator to.
++
+image::manage-subscriptions-collaboration.png[]
+
+. Once you are in the subscription management console, click the *Manage* link
+under the *Collaborators* heading, which brings you to the *Collaboration* page.
++
+image::manage-collaborators.png[]
+
+. On the *Collaboration* page, enter the *Red Hat Login ID* for the user in the user
+name field and click *Add Collaborator*.
++
+You should now see the user listed under your collaborators, the time the user
+was added, and an option to remove the user from your subscription.
++
+This does not automatically grant the user any access to your projects. Access
+must be granted manually by the project owner using `oc policy` commands. For
+example, you can log in to the cluster through the CLI using your access token
```

Comment:
How is this specifically done via the web console vs. through the CLI?

Reply:
**Via the CLI:**
Add a role to a user: oc policy add-role-to-user \<role-name\> \<username\>
Remove a role from user: oc policy remove-role-from-user \<role-name\> \<username\>
Common examples for \<role-name\> could be view, edit, and admin
CLI Docs: https://docs.openshift.com/container-platform/3.7/admin_solutions/user_role_mgmt.html#control-user-roles

**Via the Console:**
Within a project, go to Resources --> Membership
From here you can add/remove roles to/from users